keyboards/aleblazer/zodiarkv2/keyboard.json

```diff
@@ -0,0 +1,306 @@
+{
+    "keyboard_name": "Zodiark-V2",
+    "maintainer": "Aleblazer",
+    "manufacturer": "Split Logic Keyboards",
+    "url": "https://splitlogic.xyz/",
+    "usb": {
+        "vid": "0xF901",
+        "pid": "0xF930",
+        "device_version": "0.0.1"
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "nkro": true,
+        "oled": true,
+        "rgblight": false,
+        "rgb_matrix": true,
+        "mousekey": true,
+        "encoder": true
+    },
+    "diode_direction": "COL2ROW",
+	"matrix_pins": {
+        "cols": ["GP28", "GP18", "GP24", "GP22", "GP20", "GP23", "GP21"],
+        "rows": ["GP5", "GP6", "GP7", "GP8", "GP29"]
+	},
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP4", "pin_b": "GP16"},
+            {"pin_a": "GP10", "pin_b": "GP11"}
+        ]
+    },
+    "split": {
+        "enabled": true,
+        "handedness": {
+            "pin": "GP25"
+        },
+        "serial": {
+            "driver": "vendor"
+        },
+        "encoder": {
+                "right": {
+                    "rotary": [
+                        {"pin_a": "GP16", "pin_b": "GP4"},
+                        {"pin_a": "GP11", "pin_b": "GP10"}
+                    ]
+                }
+            }
+        },
+        "transport": {
+            "sync": {
+                "matrix_state": true,
+                "layer_state": true,
+                "oled": true
+            },
+            "watchdog": true
+        },
+     "bootmagic": {
+        "enabled": true,
+        "matrix": [ 0, 1 ]
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP9"
+    },
+    "rgblight": {
+        "led_count": 88,
+        "split": true,
+        "split_count": [44, 44]
+    },
```

Comment:
I see little point having RGB-Light configuration when RGB Matrix *should* be used for this keyboard.
```suggestion
```